plugins/chat/spec/serializer/chat/message_interaction_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+RSpec.describe Chat::MessageInteractionSerializer do
+  subject(:serializer) do
+    interaction = Chat::CreateMessageInteraction.call(params:, guardian: user.guardian).interaction
```

Comment:
Should we create a new fabricator with a `use_service: true` option, like we did for some other models we have in chat?

Reply:
I just didn't bother create the fabricator for now

Reply:
Up to you :slightly_smiling_face:

Reply:
I added a basic fabricator for now